### ISSUE TITLE
Native parenchymal disorders

### ISSUE THREAD

**Issue by mgriese** [obophenotype/human-phenotype-ontology#6680]
**Preferred term label:**
Native parenchymal disorders

**Synonyms**
Disorders native to the lung parenchyma

**Definition (free text, please give PubMed ID)**
Disorders originationg from the parenchymal of the lungs, manifesting over the life time, i.e. before, at, after birth during childhood, adolescence, adulthood or senescence. 

**Parent term (use hpo.jax.org/app)**
HP:0006530
Abnormal pulmonary Interstitial morphology

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 178620, OMIM: 601615, OMIM: 178640

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
see #6679

**Comment by mgriese** [comment]
It is not a clinical course, so it is not temporal; when I look to the other modifiers it also does not fit well either. 
It is more related to the cause / origin of a condition, i.e. systemic, exogenous/environmental, developmental (such are also not well established in HPO; I stopped there to continue later)

**Comment by pnrobinson** [comment]
I think this is something that should go into MONDO rather than HPO. We are discussing this in the MONDO meetings...
